### PR TITLE
Add OAuth2 public key

### PR DESCRIPTION
**Pull request by Reception123**
fixes T7386

### CODE REVIEW

**Comment by The-Voidwalker** [comment]
Probably shouldn't be pointing to a file in your home directory :)

**Comment by Reception123** [comment]
> Probably shouldn't be pointing to a file in your home directory :)

Heh yes, that was just temporary until I figured things out, this is not to be merged yet

File: OAuth2-publickey.key.pub [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDryR2aRBXgX4xvdYjP3AvECVxrBwgIn43Q/hfGM/kzqnr/ayuANpImWNdj7aJcf9HIano/7zb93uof7YocnbKw+MeG6pL5U4afQJLAWJA+PitB7UczhjZ/ybKnpU5ct6giiQum0d1RF9jYz/IVb8K9GQoB/A6T7nLDMFIG2QD3/Z91xnc0ntWmKRc5PorIZbBluY8hO+AbhVjeuF9dwhi9NwCXm1YhwD78vtJbC7flBPcNhuwU2RpaMAyl2oE8RD2uDEwuknfdFc0Zm77yGQrzH7qwgtxExh3mXxVg/wi3VxYgoud5PD9+ovte3EXGw1bxofQaekR+grgt+QN8e5be7YasKx+SLuKtpq0ZhgwAe8VX9WH5FTrXAqdD0ADXfhiE/egBaGhHgvR6pfi+xh2phFMGh4dGh63E95F+Yt8tioNczGUEHmjRRjP6aWJlCO4ImYJm3xckpswMQg6IJYSi+/9LmkJgs5RDqPrHNZQCWxLgJjV5MmJSsWzKViz2QQnxYkLjkfJoCsbtOPmIxqtpj6TlQp9Q4Xx5PJtC1Qqn2N5e6lTwYVQM7nvglhfSK72eWWb87SYdB6Vr9qQo8OpSE3kQnhcy2VJqb5bHG5jCfCphfaLxGeU0xz3tao1wclGSbTcYzojcq1TmfMTlhFyd6nfiBcKlou4xkIe0zkyaMw==
```
Comment (Universal-Omega) [inline OAuth2-publickey.key.pub:1]: Named file wrong.

Comment (Reception123) [reply OAuth2-publickey.key.pub:1]: Just fixe,d thanks!

Comment (Universal-Omega) [reply OAuth2-publickey.key.pub:1]: No problem.

**Pull request merged**